### ISSUE TITLE
Is it possible to bold text on EmailShareButton body component ? 

### ISSUE THREAD

**Issue by chea1tei** [Bunlong/next-share#74]
I'm using the EmailShareButton component body but i can't seem to bold the text.. its outputting [object Object] instead..

const body =
    <h1>Heading</h1> + '\n' +
    job?.description.replace(/<[^>]*>?/gm, '').replace('&nbsp;', ' ').slice(0, 100) + '... \n\n';

EmailShareButton
      url={`${siteUrl}/${urlPrefix}/${post?.url_slug}`}
      subject={'Our website | ' + post?.title}
      body={body} >
      <EmailIcon size={42} round />
  </EmailShareButton>

![Screenshot 2023-02-27 at 10 01 15 AM](https://user-images.githubusercontent.com/15363982/221457592-bd32146b-8781-4449-8e11-d1041ead0663.png)

**Comment by domanskyi** [comment]
Did you find any solution?

**Comment by joemcbroom** [comment]
No, since the `EmailShareButton` just generates a `mailto` link with the body as a parameter.  HTML tags are not supported.

https://stackoverflow.com/a/21010507/7922166
